ninja_extra/conf/settings.py

```diff
@@ -2,8 +2,7 @@
 
 from django.conf import settings as django_settings
 from django.core.signals import setting_changed
-from ninja import Schema
-from pydantic import Field, root_validator, validator
+from pydantic.v1 import BaseModel, Field, root_validator, validator
```

Comment:
I guess the idea to still keep pydantic V1 is to upgrade in two times.

Reply:
Yes. There is an issue with how I prep schema data before they are set to the settings schema. So I will look into that next in a different PR

Reply:
👍🏾